youtube_dl/YoutubeDL.py

```diff
@@ -178,6 +178,9 @@ class YoutubeDL(object):
     writeannotations:  Write the video annotations to a .annotations.xml file
     writethumbnail:    Write the thumbnail image to a file
     write_all_thumbnails:  Write all thumbnail formats to files
+    writestoryboard:   Write storyboard (images that appear when
+                       hovering the cursor over a video timeline) to a file
```

Comment:
-`images that appear when hovering the cursor over a video timeline`
This may not apply to other sites.

Reply:
"also known as timelines"?